### PR TITLE
Signup/Login with Parse

### PR DESCRIPTION
**Pull request by Rebecca-CYLi**
implemented Sign up/Login screens with Parse

- For next screen in sign up: need to redirect button to correct segue

- For the next screen in Login : homescreen?/any other screen

### CODE REVIEW

**Review by philipjpark — APPROVED** [review]

File: GymSwift/GymSwift/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1,24 +1,237 @@
...
+                                <nil key="textColor"/>
+                                <nil key="highlightedColor"/>
+                            </label>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" fixedFrame="YES" text="Password" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="RZt-4J-1Si">
+                                <rect key="frame" x="53" y="438" width="74" height="21"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                <nil key="textColor"/>
+                                <nil key="highlightedColor"/>
+                            </label>
```
Comment (philipjpark) [inline GymSwift/GymSwift/Base.lproj/Main.storyboard:104]: The `Secure Text Entry` box should be ticked.

File: GymSwift/GymSwift/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -1,24 +1,237 @@
...
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" fixedFrame="YES" text="Password" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="Nfl-P1-OXT">
+                                <rect key="frame" x="79" y="406" width="74" height="21"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                <nil key="textColor"/>
+                                <nil key="highlightedColor"/>
+                            </label>
```
Comment (philipjpark) [inline GymSwift/GymSwift/Base.lproj/Main.storyboard:32]: The `Secure Text Entry` box should be ticked.

**Review by philipjpark — COMMENTED** [review]
Excellent job Rebecca! I left comments that should be addressed. :)

File: GymSwift/GymSwift/Base.lproj/Main.storyboard (hunk 1 above)
Comment (Rebecca-CYLi) [reply GymSwift/GymSwift/Base.lproj/Main.storyboard:104]: fixed, done

File: GymSwift/GymSwift/Base.lproj/Main.storyboard (hunk 2 above)
Comment (Rebecca-CYLi) [reply GymSwift/GymSwift/Base.lproj/Main.storyboard:32]: fixed, done

**Pull request merged**